### ISSUE TITLE
[iOS] ShowCardWizard.json: VoiceOver focus is not moving to the control under 'Name' button after activating 'Name' button.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6417]
### Target Platforms

iOS

### SDK Version

App Version: Version 1.0 (2.3.1-beta.20210823.1)

### Application Name

Visualizer

### Problem Description

[ADO 35690892](https://microsoft.visualstudio.com/OS/_workitems/edit/35690892)

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Navigate to "V1.3" control and select it.
4. Navigate on  "V1.3" screen and select "Scenarios" button.
5. 'Scenario' screen will appear.
6. Navigate to "ShowCardWizard.json" control and select it.
7. Navigate the 'ShowCardWizard.json' screen and eanble Voiceover for the ease of access.
8. Navigate to 'Name' button and activate it.
8. Try to navigate to the newly opened card and controls using swipe gesture and observe the issue.

**Actual Result:**
VoiceOver focus is not moving to the control under 'Name' button after activating 'Name' button. Voiceover focus remains stuck on 'Name' button.

**Expected Result:**
Voiceover focus should not stuck anywhere and all the content and controls should be accessible using swipe gestures with voiceover.

### Card JSON

```JSON
ShowCardWizard.json
```

**Comment by vagpt** [comment]
Reactivating this issue as this issue is still repro on the below environment.

Test Environment:
Device: iPhone 11
iOS: 15.5
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.20220612.1)
Screen Reader: Voice Over
App Version: Version 1.0 (2.3.1-beta.20220620.1)

https://user-images.githubusercontent.com/95913748/176501726-8c1e8a56-a8ca-4ce0-9709-486c74860501.MP4

**Comment by jwoo-msft** [comment]
![ezgif com-gif-maker](https://user-images.githubusercontent.com/4112696/176511022-9922be76-9fd0-4da6-8150-044091d6d5d7.gif)

can you try this build?
https://appcenter.ms/orgs/Adaptive-Cards/apps/Adaptive-Cards-iOS-Visualizer/distribute/releases/141

**Comment by MohitTyagi13** [comment]
Issue is fixed and closed already.

Verified in latest adaptive card build Version 1.0 (2.4.0-beta.20220718.1)
